### DISCUSSION TITLE
sending a Mail doesn´t work

### DISCUSSION THREAD

**Discussion by mx2fr** [requarks/wiki#3018] [Help / Questions]
**Describe the bug**
Mailserver doesn´t work

**To Reproduce**
1. Go to 'Administration/Mail'
2. Click on 'send Test Mail'
3. See error **_Unexpected token < in JSON at position 0_**

**Screenshots**
![image](https://user-images.githubusercontent.com/48054902/105866604-72589900-5ff4-11eb-9a1a-3807068d47e3.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu Server 20.04
 - Wiki.js version: 2.5.170
 - Database engine: PostgreSQL 12.5

**Comment by NGPixel** [comment]
Post the settings you're using (masking any sensitive info obviously...)
